frontend/src/app/components/wp-table/wp-table.directive.html

```diff
@@ -54,17 +54,20 @@
       </tr>
       </tbody>
       <tbody class="results-tbody work-package--results-tbody"></tbody>
+
+      <tbody>
+        <tr class="wp-table--sums-row wp-table--row"
+            wpTableSumsRow
+            [wpTableSumsRow-table]="workPackageTable">
+
+        </tr>
+      </tbody>
```

Comment:
This moves down the "Create new work package" link regardless of whether the table is summed up:

![image](https://user-images.githubusercontent.com/617519/95214473-39e84180-07f0-11eb-91e8-e5b1d8d97793.png)

The added column for the new work package is also moved down and becomes separated from the table:

![image](https://user-images.githubusercontent.com/617519/95214563-51bfc580-07f0-11eb-9c2c-913ad590c38f.png)

Reply:
Because the string "Total sum" is longer then the string "Sum", the fist column, the one for sorting, is increased considerably in width once the total sum is displayed:

![image](https://user-images.githubusercontent.com/617519/95215315-24bfe280-07f1-11eb-9e0b-7a0caaba7557.png)

I do not know a way how to avoid this, though, as we may need to display a sum in the first column after the sorting. But I do fear the german translation already which might be "Gesamtsumme".

Reply:
Hi @ulferts ,

The white space above the 'Create wp' button has been removed.

Regarding the title's width, the only solution that I found working was to add an arbitrary max-width (55px) with tex-overflow: ellipsis. Let me know what you think.

![image](https://user-images.githubusercontent.com/25689432/103072150-54e37a00-45c5-11eb-9879-7c093956b9f8.png)

Best